lib/decidim/cdtb/spam/user_spam_detector.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require "csv"
+
+module Decidim
+  module Cdtb
+    module Spam
+      # Detect spam behavior in users
+      #
+      class UserSpamDetector < ::Decidim::Cdtb::Task
+        include ActiveSupport::Configurable
+
+        URL_REGEX = %r{(http|ftp|https)://([\w_-]+(?:(?:\.[\w_-]+)+))([\w.,@?^=%&:/~+#-]*[\w@?^=%&/~+#-])}.freeze
```

Comment:
Still not working, even escaping double slashes: `(http|ftp|https):\/\/([\w_-]+(?:(?:\.[\w_-]+)+))([\w.,@?^=%&:/~+#-]*[\w@?^=%&/~+#-])`  see https://rubular.com/r/iLWUgZrwb35pBb

Reply:
```
irb(main):001:0> URL_REGEX = %r{(http|ftp|https)://([\w_-]+(?:(?:\.[\w_-]+)+))([\w.,@?^=%&:/~+#-]*[\w@?^=%&/~+#-])}.freeze
=> /(http|ftp|https):\/\/([\w_-]+(?:(?:\.[\w_-]+)+))([\w.,@?^=%&:\/~+#-]*[\w@?^=%&\/~+#-])/
irb(main):002:0> email = "user@example.com"
=> "user@example.com"
irb(main):003:0> email =~ URL_REGEX
=> nil
irb(main):004:0> !!(email =~ URL_REGEX)
=> false
irb(main):005:0> url = "http://example.org"
=> "http://example.org"
irb(main):006:0> !!(url =~ URL_REGEX)
=> true
```

Reply:
still not working... https://rubular.com/r/jIE5CWSnjJKhdX

Reply:
yo can try with: https://rubular.com/r/6EaLDcTjBZHd95

Reply:
Good!!!! :partying_face: